tests/core/rw/rw.cpp

```diff
@@ -430,6 +430,95 @@ TEST_F(rw, trunc)
 
 	ASSERT_EQ(pmemfile_unlink(pfp, "/file1"), 0);
 
+	/* check ftruncate */
```

Comment:
Please move this block to separate test ("ftruncate"?). The missing tests for "truncate" should also be in a separate test.